cdk-lambda-layer-example/lib/cdk-lambda-layer-example-stack.ts

```diff
@@ -0,0 +1,31 @@
+import { Stack, StackProps } from 'aws-cdk-lib';
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import { Datadog } from "datadog-cdk-constructs-v2";
+import * as sst from "@serverless-stack/resources";
+import { Construct } from "constructs";
+// import { Api, StackContext } from "sst/constructs";
+
+export class CdkLambdaLayerExampleStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const datadog = new Datadog(this, "Datadog", {
+      nodeLayerVersion: 87,
+      extensionLayerVersion: 38,
+      site: "datadoghq.com",
+      apiKeySecretArn: "xyz"
+    });
+
+    const exampleLambda = new lambda.Function(this, 'LayerExampleLambda', {
+      functionName: 'CodeSee-Lambda-Layer-Example',
+      code: lambda.Code.fromAsset('./src'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_16_X,
+      layers: [
+        lambda.LayerVersion.fromLayerVersionArn(this, 'CodeSeeLayer', 'arn:aws:lambda:us-east-2:666523192759:layer:codesee-dd-bridge:8'),
+      ]
+    });
+
+    datadog.addLambdaFunctions([exampleLambda])
```

Comment:
This looks a lot like what we tried earlier today, but there was no Datadog Layer on the deployed lambda. Did you verify both layers are present? Do you have any idea why we were seeing only the CodeSee layer before?

Reply:
tagged you in the thread https://codesee.slack.com/archives/C04SEN84VCG/p1677629816001899?thread_ts=1677622945.512659&cid=C04SEN84VCG

Reply:
Verified both layers are present. No idea why we are not seeing, probably sst framework conflict, I'll check that next.